frontend/device/kobo/nickel_conf.lua

```diff
@@ -143,4 +155,12 @@ function NickelConf.frontLightState.set(new_state)
                                        true)
 end
 
+function NickelConf.colorSetting.set(new_color)
+    assert(new_color >= 1500 and new_color <= 6400,
```

Comment:
Note that asserts and errors generally speaking mean insta-crash. As such we want to restrict them to debug-mode only, see

https://github.com/koreader/koreader/blob/775b7a364f4b0a1b82fc236d4aaa6513922f9904/frontend/ui/uimanager.lua#L298-L302

You can also do things with if dbg.is_on but that doesn't seem appropriate here (nor does pcall).